ha/const.py

```diff
@@ -244,6 +246,7 @@ class ALERT_ATTRIBUTES:
     MODULE = "module"
     EVENT = "event"
     IEC = "IEC"
+    logger_utility_iec_cmd="logger -i -p local3.err"
```

Comment:
This is not an alert attribute, so we can put it outside of this enum

Reply:
Ok.